.github/workflows/docker-dev-release.yml

```diff
@@ -77,24 +87,27 @@ jobs:
           load: true  # Load the build images into the local docker.
       - name: Test Image
         run: |
-          image_tag=${{ steps.metadata.outputs.tags }}
           echo ${{ steps.build.outputs.digest }}
-          image_digest=${{ env.image }}@${{ steps.build.outputs.digest }}
-          
+          image_tags=(${{ steps.metadata.outputs.tags }})
+
           # install redis-tools
           sudo apt-get install redis-tools -y
-          
-          docker image inspect ${image_tag}
-          echo "Testing ${{ matrix.flavor }} image"
-
-          # docker run with port-forwarding
-          docker run -d -p 6379:6379 ${image_tag}
-          sleep 5
-          if [[ $(redis-cli -h localhost ping) != "PONG" ]]; then
-            echo "Redis ping failed"
-            exit 1
-          fi
-          echo "Redis ping succeeded"
+
+          for image_tag in "${image_tags[@]}"; do
```

Comment:
really not an expert. can we build images like before and just create and push manifests to both registries in the end?
why do we need to change anything here?

Reply:
Yep, I didn't change anything much here. As the image now has two tags (one for ghcr and another for artifact registry), this step ensures that the image is pushed to both registries. So I just wrapped the previous code in for loop to test.